### PR TITLE
Added a getting started page

### PR DESCRIPTION
**Pull request by fraincs**
Added a getting-started page under /pages/getting-started this is a temporary URL until the components page are released allowing us to publish the homepage that will link to the getting-started page

### CODE REVIEW

File: apps/docs/content/pages/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+</HopperProvider>
+```
+
+<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout>
+
+## Detailed Installation
+
+### Components
+
+Hopper Components are React Comtponents, uf you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:30]:
```suggestion
Hopper Components are React Components, if you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developing your own components.
```

File: apps/docs/content/pages/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+##### Fonts
+
+In order to use Hopper fonts in your project, import the following:
+
+```css
+@import "@hopper-ui/tokens/fonts.css";
+```
+
+All the necessary font-face declarations are now imported in your project.
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:90]:
```suggestion
All the necessary font-face declarations are now imported into your project.
```

File: apps/docs/content/pages/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+<PackageInstallation library="icons @hopper-ui/styled-system react-aria-components" mod="prod" />
+
+Import the styles in your project:
+
+```css
+@import "@hopper-ui/icons/index.css";
+@import "@hopper-ui/styled-system/index.css";
+```
+
+Configuring your application it a little different:
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:145]:
```suggestion
Configuring your application is a little different:
```

File: apps/docs/content/pages/getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+```tsx
+import { AddIcon } from "@hopper-ui/icons";
+
+<AddIcon />
+```
+
+##### React 16 Installation
+
+If your'e still using React 16, you need to use the `@hopper-ui/icons-react16` package.
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:171]:
```suggestion
If you're still using React 16, you need to use the `@hopper-ui/icons-react16` package.
```

File: apps/docs/content/pages/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+<HopperProvider colorScheme="light">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout>
+
+## Detailed Installation
+
+### Components
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:28]: I've noticed that there is not much difference in style/size between h3-4. I feel like it ends up being a bit messy or hard to follow. Not sure if this would be something easy to fix or not.

Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:28]: The main issue is that h5 were so small that I bumped them, I'll see what can be done

Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:28]: Here's the current situation: 
h5: 16px
h4: 18px
h3: 20px
h2: 24px
h1: 40px

If h3 goes to 24, h2 would need in theory need to go to 32px its big and close to the h1

image

Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:28]: We are using the system fonts, therefore we don't have much control over boldness, as we can't assume a system font will support a semi bold.

File: apps/docs/content/pages/getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:12]: image

weird looking in dark mode

File: apps/docs/content/pages/getting-started.mdx [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+## Detailed Installation
+
+### Components
+
+Hopper Components are React Comtponents, uf you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
+
+#### Installation
+
+Run one of the following commands in your terminal to install the Hopper Components package.
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:34]: There is only one command under. I understand you are refering to either npm/pnpm/yarn, but there is 1 command, for 3 different tools. I would phrase it like 

```
Run the following command in your terminal to install the Hopper packages.
```

File: apps/docs/content/pages/getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:15]: manque la section où on doit importer les fichier CSS

File: apps/docs/content/pages/getting-started.mdx (hunk 6 above)
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:12]: For normal react installations, importing @hopper-ui/tokens is not required

File: apps/docs/content/pages/getting-started.mdx [hunk 9]
```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:6]: I'm not sure how to word this properly, but i feel like this page is really really too techy, and not consumer oriented. 

There is no such thing as a "Quick Install". 

There are 2 main use case: 
- You're a react dev
- You're a non react dev.

It should be reflected straight away, and maybe redirected to a new page, such as getting-started/react. 

So `/getting-started`

This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.

# Installation Guide

Follow one of the guides below to get started with Hopper. 

[ React Application ] [ JavaScript Application ] [ Individual Packages Installation]

# Contribute

Pull requests are welcome. For major changes, please open a [discussion](https://github.com/gsoft-inc/wl-hopper/discussions/new/choose) first to discuss what you would like to change. If you're interested in contributing, check out our [Contributing Guide](https://github.com/gsoft-inc/wl-hopper/blob/main/CONTRIBUTING.md)!
  1 change: 1 addition & 0 deletions1  
[apps/docs/package.json](https://github.com/gsoft-inc/wl-hopper/pull/254/files#diff-d03d2ce97fad5059ad6248fca5f5919994b09bca360f883318c03f1f817ae677)

Then `/getting-started/react`

<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how. LINK TO INDIVIDUAL PACKAGE INSTALLATION REACT 16 ICONS</Callout>

La section qui est dans quick install ici


Then `/getting-started/javascript`

la section tokens + svg-icons mergé

Then `/getting-started/individual-packages`

tt les installations

File: apps/docs/content/pages/getting-started.mdx [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+
+```tsx
+import { AddIcon } from "@hopper-ui/icons";
+import { ConversationRichIcon } from "@hopper-ui/icons";
+
+<AddIcon />
+<ConversationRichIcon />
+```
+
+#### Non-standard installations
```
Comment (franckgaudin) [inline apps/docs/content/pages/getting-started.mdx:110]: Non-standard is a bit of a strange name, I don't have much inspiration for a title. Maybe **Javascript installations** or **Framework agnostic installations**.

File: apps/docs/content/pages/getting-started.mdx [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+
+Alternatively, you can utilize SVG icons within your CSS in the following manner:
+
+```css
+.myComponent {
+    background-image: url("@hopper-ui/svg-icons/alert-24.svg");
+}
+```
+
+##### React Icons Standalone Installation
```
Comment (franckgaudin) [inline apps/docs/content/pages/getting-started.mdx:132]: I'd remove React Icons because it's redundant 

```suggestion
##### Standalone Installation
```

And I don't quite understand the purpose of this section, isn't it a light version of using Hopper?

File: apps/docs/content/pages/getting-started.mdx [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+##### React 16 Installation
+
+If your'e still using React 16, you need to use the `@hopper-ui/icons-react16` package.
+
+<PackageInstallation library="icons-react16" mod="prod" />
+
+The usage is the same as the default package, you only need to import the icons from a different package.
+
+```tsx
+import { AddIcon } from "@hopper-ui/icons-react16";
```
Comment (franckgaudin) [inline apps/docs/content/pages/getting-started.mdx:178]: here, too, the elements can be grouped 
```suggestion
import { AddIcon, ConversationRichIcon } from "@hopper-ui/icons-react16";
```

File: apps/docs/content/pages/getting-started.mdx [hunk 13]
```diff
@@ -0,0 +1,187 @@
...
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
```
Comment (franckgaudin) [inline apps/docs/content/pages/getting-started.mdx:16]: I'll give a slightly more specific example for here, as we mention using the Provider at application root.

```
import { HopperProvider } from "@hopper-ui/components";
import { createRoot } from 'react-dom/client';

const root = createRoot(document.getElementById('root'));
root.render(
  < HopperProvider >
    <App />
  </HopperProvider >
);
```

File: apps/docs/content/pages/getting-started.mdx (hunk 6 above)
Comment (franckgaudin) [inline apps/docs/content/pages/getting-started.mdx:12]: Packages will be missing `react-aria` `react-aria-components`

File: apps/docs/content/pages/getting-started.mdx (hunk 9 above)
Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:6]: I agree to an extent, I feel that adding "hub" page is an unnecessary click in most cases. How about /getting-started would be the default installation aka react?

Comment (alexasselin008) [reply apps/docs/content/pages/getting-started.mdx:6]: sure ya

File: apps/docs/content/pages/getting-started.mdx (hunk 11 above)
Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:132]: @alexasselin008 can you confirm this?

Comment (alexasselin008) [reply apps/docs/content/pages/getting-started.mdx:132]: if it can simplify this page we can remove it. But there needs to be orbiter or hopper components installed with the HopperPRovider/ThemePRovider for the icons to work properly

File: apps/docs/content/pages/getting-started.mdx [hunk 14]
```diff
@@ -0,0 +1,44 @@
+---
+title: Getting started
+description: Getting started with Hopper UI.
+---
+
+{/* <Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout> */}
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:6]: is this commented out? how come?

File: apps/docs/content/pages/getting-started.mdx [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+
+This guide will help you getting started with Hopper UI. Hopper UI is a Design System that provides a collection of components, [tokens](/tokens), and [icons](/icons) to help you build applications.
```
Comment (vicky-comeau) [inline apps/docs/content/pages/getting-started.mdx:8]:
```suggestion
This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, [tokens](/tokens), and [icons](/icons) to help you build applications.
```

File: apps/docs/app/pages/getting-started/page.tsx
Comment (franckgaudin) [inline apps/docs/app/pages/getting-started/page.tsx:1]: As you have pages related to this section, I would display the navigation in addition to the internal content links. 

image

File: apps/docs/content/pages/javascript.mdx [hunk 16]
```diff
@@ -0,0 +1,68 @@
+---
+title: Javascript
+description: An installation guide for Hopper UI for non-React users.
+---
+
+If you are a non-React user you can still benefit from Hopper UI. This guide will help you get started with Hopper UI.
+
+## Tokens
+
+Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](/tokens/semantic/color) are available for everyone, where components tokens are designed with non React users in mind.
```
Comment (vicky-comeau) [inline apps/docs/content/pages/javascript.mdx:10]:
```suggestion
Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](/tokens/semantic/color) are available for everyone, where component tokens are designed with non React users in mind.
```

File: apps/docs/app/pages/getting-started/individual-packages/page.tsx [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    const sectionLinks = getSectionLinks(page);
+
+    return (
+        <div className="hd-wrapper hd-flex">
+            <Aside title="On this page" links={sectionLinks} />
+            <main>
+                <article className="hd-content" key={page._id}>
+                    <h1 className="hd-title hd-title--level1">{page.title}</h1>
```
Comment (franckgaudin) [inline apps/docs/app/pages/getting-started/individual-packages/page.tsx:23]: The `Title` component exists for this use case. 

`<Title as="h1">{title}</Title>`

ref: https://github.com/gsoft-inc/wl-hopper/blob/6cd15ebe9e25582349fb519cd22f01d6e52fa22a/apps/docs/components/title/Title.tsx

File: apps/docs/app/pages/getting-started/javascript/page.tsx [hunk 18]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    const sectionLinks = getSectionLinks(page);
+
+    return (
+        <div className="hd-wrapper hd-flex">
+            <Aside title="On this page" links={sectionLinks} />
+            <main>
+                <article className="hd-content" key={page._id}>
+                    <h1 className="hd-title hd-title--level1">{page.title}</h1>
```
Comment (franckgaudin) [inline apps/docs/app/pages/getting-started/javascript/page.tsx:23]: Use the `Title` component

https://github.com/gsoft-inc/wl-hopper/blob/6cd15ebe9e25582349fb519cd22f01d6e52fa22a/apps/docs/components/title/Title.tsx

File: apps/docs/app/pages/getting-started/page.tsx [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    const sectionLinks = getSectionLinks(page);
+
+    return (
+        <div className="hd-wrapper hd-flex">
+            <Aside title="On this page" links={sectionLinks} />
+            <main>
+                <article className="hd-content" key={page._id}>
+                    <h1 className="hd-title hd-title--level1">{page.title}</h1>
```
Comment (franckgaudin) [inline apps/docs/app/pages/getting-started/page.tsx:23]: Use `Title` component

https://github.com/gsoft-inc/wl-hopper/blob/6cd15ebe9e25582349fb519cd22f01d6e52fa22a/apps/docs/components/title/Title.tsx

**Review by franckgaudin — COMMENTED** [review]
The page's internal navigation doesn't sit well on mobile devices.

Capture d’écran, le 2024-06-04 à 09 34 18

File: apps/docs/content/pages/getting-started.mdx [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+import { createRoot } from 'react-dom/client';
+
+const root = createRoot(document.getElementById('root'));
+root.render(
+  < HopperProvider >
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:27]: Weird spacing it should be 
```
  <HopperProvider>
    <App />
  </HopperProvider>
```

File: apps/docs/content/pages/getting-started.mdx [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+    <App />
+  </HopperProvider >
+);
+```
+
+You will also need to import the Hopper styles in your project. You can do this by importing the following CSS files:
+
+```css
+@import "@hopper-ui/tokens/tokens.css";
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:36]: c'est pas ça le CSS à importé, c'est 

```
@import "@hopper-ui/components/index.css";
@import "@hopper-ui/icons/index.css";
```

File: apps/docs/content/pages/getting-started.mdx [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+Below is the typical installation path, this assumes you are using React. If you have particluar needs you can have a look at these pages:
+
+- [Javascript installation](/pages/getting-started/javascript)
+- [Individual package installation](/pages/getting-started/individual-packages)
+
+Run the following command in your terminal to install the Hopper packages.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/styled-system react-aria react-aria-components" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
```
Comment (alexasselin008) [inline apps/docs/content/pages/getting-started.mdx:19]: This page would benefit of more headings, similar to orbit

File: apps/docs/content/pages/getting-started.mdx (hunk 14 above)
Comment (fraincs) [reply apps/docs/content/pages/getting-started.mdx:6]: Good call, I thought it was not necessary anymore with the way I restructured things

File: apps/docs/content/getting-started/react.mdx [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/styled-system react-aria react-aria-components" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+import { createRoot } from 'react-dom/client';
+
+const root = createRoot(document.getElementById('root'));
```
Comment (alexasselin008) [inline apps/docs/content/getting-started/react.mdx:26]: you closed 2 of my comments but the 2 issues are still there: 
https://github.com/gsoft-inc/wl-hopper/pull/254#discussion_r1626269327
https://github.com/gsoft-inc/wl-hopper/pull/254#discussion_r1626565852

File: apps/docs/content/getting-started/individual-packages.mdx [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+title: Individual Packages
+description: An how to guide to install Hopper UI packages individually.
+order: 3
+---
+
+If you need a particluar installation, you can install Hopper UI packages individually.
+
+## Components
+
+Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
```
Comment (vicky-comeau) [inline apps/docs/content/getting-started/individual-packages.mdx:11]:
```suggestion
Hopper Components are React Components. If you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developing your own components.
```
Feel free to update the sentence, but it doesn't seem right to me, they are two different sentences.

File: apps/docs/content/getting-started/javascript.mdx [hunk 25]
```diff
@@ -0,0 +1,69 @@
...
+title: Javascript
+description: An installation guide for Hopper UI for non-React users.
+order: 2
+---
+
+If you are a non-React user you can still benefit from Hopper UI. This guide will help you get started with Hopper UI.
+
+## Tokens
+
+Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](/tokens/semantic/color) are available for everyone, where components tokens are designed with non React users in mind.
```
Comment (vicky-comeau) [inline apps/docs/content/getting-started/javascript.mdx:11]:
```suggestion
Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](/tokens/semantic/color) are available for everyone, where component tokens are designed with non React users in mind.
```

File: apps/docs/content/getting-started/react.mdx [hunk 26]
```diff
@@ -0,0 +1,51 @@
+---
+title: React
+description: Getting started with Hopper UI.
+order: 1
+---
+
+This guide will help you getting started with Hopper UI. Hopper UI is a Design System that provides a collection of components, [tokens](/tokens), and [icons](/icons) to help you build applications.
```
Comment (vicky-comeau) [inline apps/docs/content/getting-started/react.mdx:7]:
```suggestion
This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, [tokens](/tokens), and [icons](/icons) to help you build applications.
```

Comment (vicky-comeau) [inline apps/docs/content/getting-started/react.mdx:7]: The links to /tokens and /icons don't actually work.

File: apps/docs/app/getting-started/[...slug]/page.tsx [hunk 27]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    const sectionLinks = getSectionLinks(pages);
+
+    return (
+        <div className="hd-container">
+            <Aside title="On this page" links={sectionLinks} />
+            <main>
+                <article className="hd-content" key={pages._id}>
+                    <h1 className="hd-title hd-title--level1">{pages.title}</h1>
```
Comment (franckgaudin) [inline apps/docs/app/getting-started/[...slug]/page.tsx:37]: Use `Title` component

https://github.com/gsoft-inc/wl-hopper/blob/6cd15ebe9e25582349fb519cd22f01d6e52fa22a/apps/docs/components/title/Title.tsx

File: apps/docs/app/pages/getting-started/javascript/page.tsx
Comment (franckgaudin) [inline apps/docs/app/pages/getting-started/javascript/page.tsx:1]: same thing here, the page has been moved to getting-started

File: apps/docs/components/callout/Callout.stories.tsx [hunk 28]
```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import Callout from "./Callout.tsx";
+
+
```
Comment (franckgaudin) [inline apps/docs/components/callout/Callout.stories.tsx:5]: remove the empty line

**Review by alexasselin008 — DISMISSED** [review]

File: apps/docs/app/getting-started/page.tsx [hunk 29]
```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { allPages } from "contentlayer/generated";
+import Mdx from "@/components/mdx/Mdx";
+import { notFound } from "next/navigation";
+
+export default function IconPage() {
+    const page = allPages.find(iconPage => iconPage._id === "pages/icons.mdx");
```
[the remark this answers is not in the document]
Comment (fraincs) [reply apps/docs/app/getting-started/page.tsx:8]: this will be renamed

File: apps/docs/app/getting-started/layout.tsx [hunk 30]
```diff
@@ -0,0 +1,32 @@
+"use client";
```
[the remark this answers is not in the document]
Comment (fraincs) [reply apps/docs/app/getting-started/layout.tsx:1]: When doing so I get this error: ./app/getting-started/layout.tsx
Error:
  × You're importing a component that needs useSelectedLayoutSegment. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by

[the remark this answers is not in the document]
Comment (fraincs) [reply apps/docs/app/getting-started/layout.tsx:1]: Icon layout has also a use client declaration

File: apps/docs/app/getting-started/[...slug]/page.tsx (hunk 27 above)
Comment (franckgaudin) [reply apps/docs/app/getting-started/[...slug]/page.tsx:36]: @fraincs  this comment has been marked as resolved but I can't see any change?

**Review by franckgaudin — APPROVED** [review]
